src/docfx/build/context/BuildScope.cs

```diff
@@ -28,18 +27,18 @@ internal class BuildScope
 
         public RedirectionMap Redirections { get; }
 
-        public BuildScope(ErrorLog errorLog, Input input, Docset docset, Docset fallbackDocset, Dictionary<string, (Docset docset, bool inScope)> dependencyDocsets, TemplateEngine templateEngine, MonikerProvider monikerProvider)
+        public BuildScope(
+            ErrorLog errorLog, Input input, DocumentProvider documentProvider, TemplateEngine templateEngine, Docset docset, Docset fallbackDocset, MonikerProvider monikerProvider)
         {
-            var config = docset.Config;
-
             _input = input;
-            _glob = CreateGlob(config);
-            _templateEngine = templateEngine;
+            _docset = docset;
```

Comment:
seems only `_docset.Config` is needed, is `_config` enough since `docset` contains more stuff?

Reply:
It is still used by `RedirectionMap` currently